client/src/components/solution-display-widget/index.tsx

```diff
@@ -27,9 +27,13 @@ export function SolutionDisplayWidget({
   const { t } = useTranslation();
 
   const showOrViewText =
-    displayContext === 'settings'
-      ? t('buttons.show-solution')
-      : t('buttons.view');
+    displayContext === 'settings' ? (
+      <Trans i18nKey='buttons.view-solution'>
+        <span className='sr-only'>Solution</span>
+      </Trans>
+    ) : (
+      t('buttons.view')
+    );
```

Comment:
with deletion of other view variation, we can have simple translated view text
```suggestion
 const viewText = t('buttons.view')
```